.github/workflows/ci.yml

```diff
@@ -23,20 +23,26 @@ jobs:
         with:
           python-version: 3.7
 
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v4
+      - name: Setup Conda with ${{ matrix.python-version }}
+        uses: conda-incubator/setup-miniconda@v2
```

Comment:
Setting up conda and memba in the CI takes longer than using `setup-python`. For a library with very few dependencies, it's not worth it. Also, the CI changes are unrelated to the topic of this PR and should be addressed separately.

Reply:
Makes sense, originally I thought of testing the new environment handler in CI as well, which is why I wanted to add it here, but I guess that could be elsewhere.